client/src/modules/user/model/actions.js

```diff
@@ -149,3 +149,6 @@ export const getAccountDetails = token =>
 
 export const deleteAccount = (email, password) =>
   deleteData('/auth', { email, password });
+
+export const saveEmailNotificationSettings = notificationFrequency =>
+  postData('/auth/email-notification-settings', { notificationFrequency });
```

Comment:
We always displayed notification here (except of auth forms). Why there are placed now in profile view? This is not consistently